tck/features/clauses/call/Call5.feature

```diff
@@ -29,3 +29,109 @@
 #encoding: utf-8
 
 Feature: Call5 - Results projection
+
+  Scenario: [1] Explicit procedure result projection
+    Given an empty graph
+    And there exists a procedure test.my.proc(in :: INTEGER?) :: (out :: STRING?):
+      | in   | out   |
+      | null | 'nix' |
+    When executing query:
+      """
+      CALL test.my.proc(null) YIELD out
+      RETURN out
+      """
+    Then the result should be, in order:
+      | out   |
+      | 'nix' |
+    And no side effects
+
+  Scenario: [2] Rename procedure call output in RETURN clause
+    Given an empty graph
+    And there exists a procedure test.my.proc(in :: INTEGER?) :: (out :: STRING?):
+      | in   | out   |
+      | null | 'nix' |
+    When executing query:
+      """
+      CALL test.my.proc(null) YIELD out
+      RETURN out as a
+      """
+    Then the result should be, in order:
+      | a     |
+      | 'nix' |
+    And no side effects
+
+  Scenario: [3] Explicit procedure result projection with RETURN *
+    Given an empty graph
+    And there exists a procedure test.my.proc(in :: INTEGER?) :: (out :: STRING?):
+      | in   | out   |
+      | null | 'nix' |
+    When executing query:
+      """
+      CALL test.my.proc(null) YIELD out
+      RETURN *
+      """
+    Then the result should be, in order:
+      | out   |
+      | 'nix' |
+    And no side effects
+
+  Scenario: [4] Project procedure results between query scopes with WITH clause
+    Given an empty graph
+    And there exists a procedure test.my.proc(in :: INTEGER?) :: (out :: STRING?):
+      | in   | out   |
+      | null | 'nix' |
+    When executing query:
+      """
+      CALL test.my.proc(null) YIELD out
+      WITH out RETURN out
+      """
+    Then the result should be, in order:
+      | out   |
+      | 'nix' |
+    And no side effects
+
+  Scenario: [5] Project procedure results between query scopes with WITH clause and rename the projection
+    Given an empty graph
+    And there exists a procedure test.my.proc(in :: INTEGER?) :: (out :: STRING?):
+      | in   | out   |
+      | null | 'nix' |
+    When executing query:
+      """
+      CALL test.my.proc(null) YIELD out
+      WITH out as a RETURN a
+      """
+    Then the result should be, in order:
+      | a     |
+      | 'nix' |
+    And no side effects
+
+  Scenario: [6] Respect the YIELD order - 1
+    Given an empty graph
+    And there exists a procedure test.my.proc(in :: INTEGER?) :: (A :: INTEGER?, B :: INTEGER?) :
+      | in   | A | B |
+      | null | 1 | 2 |
+    When executing query:
+      """
+      CALL test.my.proc(null) YIELD A, B
+      WITH out as a RETURN a
+      """
+    Then the result should be, in order:
+      | A | B |
+      | 1 | 2 |
+    And no side effects
```

Comment:
This scenario is wrong. The one below too. With `RETURN a` the result table will only have `a` as its single column.

With regards to my comment above about the order of yield items is irrelevant, I would change the scenario as follows:

```suggestion
  Scenario Outline: [6] The order of yield items is irrelevant
    Given an empty graph
    And there exists a procedure test.my.proc(in :: INTEGER?) :: (A :: INTEGER?, B :: INTEGER?) :
      | in   | a | b |
      | null | 1 | 2 |
    When executing query:
      """
      CALL test.my.proc(null) YIELD <yield>
      RETURN a, b
      """
    Then the result should be, in order:
      | a | b |
      | 1 | 2 |
    And no side effects
    
    Examples:
      | yield |
      | a, b  |
      | b, a  |
```